bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+import model SM_Ltotal_Ind5v2020v2_UFO
+
+define p = g u c d s u~ c~ d~ s~
```

Comment:
note that there is a potential issue. you might get a 5fs pdf but you want a 4fs pdf. can you try what default pdf gives you? you should also make sure that the proper pdf is in your systematics part. 
one other question: why is b part of j ?

Reply:
Hi, @agrohsje okay, we have changed it.
define p = g u c d s b u c d s b~
define j = p
Please have a look.

Reply:
Hello @agrohsje @geet59 , are there any news about the success?